bindings/python/python_constants.cpp

```diff
@@ -1051,4 +1051,28 @@ void addLabels(PyObject * dict)
     ADD_CONST(BSOFT_SYMMETRY_EQUIV_ID);
     ADD_CONST(BSOFT_SYMMETRY_EQUIV_POS_AS_XYZ);
 
+    // BUFFER labels
+    ADD_CONST(BUFFER_LABELS_START);
+    ADD_CONST(BUFFER1);
+    ADD_CONST(BUFFER2);
+    ADD_CONST(BUFFER3);
+    ADD_CONST(BUFFER4);
+    ADD_CONST(BUFFER5);
+    ADD_CONST(BUFFER6);
+    ADD_CONST(BUFFER7);
+    ADD_CONST(BUFFER8);
+    ADD_CONST(BUFFER9);
+    ADD_CONST(BUFFER10);
+    ADD_CONST(BUFFER11);
+    ADD_CONST(BUFFER12);
+    ADD_CONST(BUFFER13);
+    ADD_CONST(BUFFER14);
+    ADD_CONST(BUFFER15);
+    ADD_CONST(BUFFER16);
+    ADD_CONST(BUFFER17);
+    ADD_CONST(BUFFER18);
+    ADD_CONST(BUFFER19);
+    ADD_CONST(BUFFER20);
+
```

Comment:
Since they should be used in the scope of a process there is no risk of overlapping....BSOFT can safely make use of the same buffer labels...unless there is a process that mixes both cases (can't think of any reasonable one).

Reply:
Ok, great!